dashboards/mongodb-prometheus/mongodb.json

```diff
@@ -0,0 +1,1714 @@
+{
+  "name": "MongoDB Prometheus",
```

Comment:
For coherence with other Prometheus integrations I would add parenthesis to Prometheus : MongoDB (Prometheus)

Reply:
👍